xarray/core/dataarray.py

```diff
@@ -486,6 +486,11 @@ def _attr_sources(self):
         """List of places to look-up items for attribute-style access"""
         return [self.coords, LevelCoordinatesSource(self), self.attrs]
```

Comment:
Should `self.dims` also be in this list? I think so?

Reply:
You are right.
I fixed this and added tests for this.